src/main/api/allApi.js

```diff
@@ -0,0 +1,7 @@
+const API_BASE_URL_FOLDER_ALL = 'https://bootcamp-api.codeit.kr/api/users/1/links';
+
+export async function getAllFolders() {
+  const response = await fetch(API_BASE_URL_FOLDER_ALL)
+  const datas = await response.json();
+  return datas;
```

Comment:
```suggestion
  return await response.json();
```
[중요도: 중간] 굳이 상수를 선언해서 return할 필요는 없어보입니다.

Reply:
오호... 이렇게 해도 되네요 감사합니다!

Reply:
네 좋습니다! 사실 .json() 함수가 비동기가 아닌지라 await을 빼도 되고, 굳이굳이 재선언할 필요는 없는거니까요~